services/api/mw2pdf/src/index.ts

```diff
@@ -23,6 +24,7 @@ program
   .option('--subtitle <string>', 'Subtitle of book', 'Table of Contents')
   .option('--mwUsername <string>', 'The username to log in with', '')
   .option('--mwPassword <string>', 'The password to log in with', '')
+  .option('--pageSize <string>', 'The target page size for the book', 'Letter')
```

Comment:
And now we have the last possibility remaining: "Letter"!  Which I guess makes sense, because this is for the human-readable display.